website/pages/components/toggle-group/index.tsx

```diff
@@ -0,0 +1,21 @@
+import Head from "next/head";
+import type { ReactElement } from "react";
+import ToggleGroupPageLayout from "../../../screens/components/toggle-group/ToggleGroupPageLayout";
+import ToggleGroupUsagePage from "../../../screens/components/toggle-group/usage/ToggleGroupUsagePage";
+
+const Index = () => {
+  return (
+    <>
+      <Head>
+        <title>ToggleGroup — Halstack Design System</title>
+      </Head>
```

Comment:
I would change the title to `Toggle Group`, like for `Text Input`, `Number Input` and the rest of components with more than one word in its title.